emcc.py

```diff
@@ -1429,6 +1444,22 @@ def save_intermediate(name=None, suffix='js'):
       file_code = execute([shared.PYTHON, shared.FILE_PACKAGER, unsuffixed(target) + '.data'] + file_args, stdout=PIPE)[0]
       pre_js = file_code + pre_js
 
+    if shared.Settings.BINARYEN:
+      # add in the glue integration code as a pre-js, so it is optimized together with everything else
+      wasm_js_glue = open(os.path.join(shared.BINARYEN_ROOT, 'src', 'js', 'wasm.js-post.js')).read()
+      wasm_js_glue = wasm_js_glue.replace("Module['asmjsCodeFile']", '"' + os.path.basename(asm_target) + '"') # " or '? who knows :)
```

Comment:
Right, thanks - this is a relic of this code being injected in another way, but now that things are sane, I can fix it.